### PR TITLE
[Doc Improvement][Apps support update for channel meetings]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3a6d4bd052be9bd84b881f9048273f3624afcecf/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/498eeca4-4421-4c03-a3e0-c4a75a02bd60) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/023be76a0302033850ea6a24b6b61a7159f99b77/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2228fffd-43c9-4249-9d05-c407210f2ba1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3d4189e1277a83bcc3565f789d3f9003e284e124/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ae9de85-417f-42f7-8c21-4f125e8bd492) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 1]
```diff
@@ -16,7 +16,12 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 >
 > * Apps for scheduled public channel meetings are currently available only in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
 >
-> * Apps aren't supported in [Public Switched Telephone Network (PSTN)](/microsoftteams/cloud-voice-landing-page#public-switched-telephone-network-connectivity-options) and [end-to-end encrypted Teams calls](https://support.microsoft.com/office/use-end-to-end-encryption-for-teams-calls-1274b4d2-b5c5-4b24-a376-606fa6728a90).
+> * Apps aren't supported in the following:
+>
```
Comment (fallothakor) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:20]: Please remove this line space.

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 2]
```diff
@@ -16,7 +16,12 @@ Meetings enable collaboration, partnership, informed communication, and shared f
...
+>   * [Public Switched Telephone Network (PSTN) Teams calls](/microsoftteams/cloud-voice-landing-page#public-switched-telephone-network-connectivity-options)
+>   * [End-to-end encrypted Teams calls](https://support.microsoft.com/office/use-end-to-end-encryption-for-teams-calls-1274b4d2-b5c5-4b24-a376-606fa6728a90)
+>   * [Instant channel meetings](https://support.microsoft.com/office/start-an-instant-meeting-in-teams-ff95e53f-8231-4739-87fa-00b9723f4ef5)
+>   * [Shared channel meetings](https://support.microsoft.com/office/schedule-a-meeting-in-teams-943507a9-8583-4c58-b5d2-8ec8265e04e5#ID0EFBD=Desktop)
```
Comment (fallothakor) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:24]: this is not correct reference link, pls link to documentation related to meeting in shared channels. fyi, I couldnt find one, so, if that the case for you, May be reword it to Meetings in Shared Channel and link shared channel keyword to appropriate doc.

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 3]
```diff
@@ -16,7 +16,12 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 >
```
[the remark this answers is not in the document]
Comment (fallothakor) [reply msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:16]: also, please remove line # 44 its duplicate now. See if you think notes on #40 now needs to be merged with these notes.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/95a3f53b426d825ac7dedb0d6228d86be1dae894/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4f248e3-177c-4c1a-a4d1-c454f13f20e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/81b671833d502b78a715c008dbdbc779cd47f26c/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/77f6a53a-71dc-4e59-a0ea-dd9fc2e5f58f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by fallothakor — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e1d6ad1e613272910fb323f15fcc5c703f6cbf7d/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d6dfd79-3322-4527-899a-4946d2a89b9f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
